### PR TITLE
ENG-962 Improvements on Jira to Cal demo

### PR DESCRIPTION
**Pull request by pashafateev**
Related to this [PR](https://github.com/autokitteh/autokitteh/pull/433) without which this code will not run.

### CODE REVIEW

File: jira_to_google_calendar/program.py [hunk 1]
```diff
@@ -16,39 +16,25 @@
...
-    desc = f"Link to Jira Issue: {base_url}/browse/{issue.key}\n\n"
-    issue_details = autokitteh.AttrDict(
-        {
-            "description": desc + issue.fields.description,
-            "duedate": issue.fields.duedate,
-            "summary": issue.fields.summary,
-        }
-    )
-    return issue_details
+    _create_calendar_event(event.data.issue.fields, event.data.issue.key)
```
Comment (daabr) [inline jira_to_google_calendar/program.py:25]: Short and sweet, cool! Please just remove the empty line above.

File: jira_to_google_calendar/program.py [hunk 2]
```diff
@@ -16,39 +16,25 @@
...
 
 
 @autokitteh.activity
-def _create_calendar_event(issue_details):
-    start_time = datetime.strptime(issue_details.duedate, "%Y-%m-%d")
+def _create_calendar_event(issue, key):
+    start_time = datetime.strptime(issue.duedate, "%Y-%m-%d")
     end_time = start_time + timedelta(minutes=30)
+    url = get_url(os.getenv("JIRA_CONNECTION_NAME"))
```
Comment (daabr) [inline jira_to_google_calendar/program.py:32]:
```suggestion
    url = get_base_url(os.getenv("JIRA_CONNECTION_NAME"))
```

**Review by daabr — APPROVED** [review]
Just a few cosmetic notes

File: jira_to_google_calendar/program.py [hunk 3]
```diff
@@ -16,39 +16,25 @@
 
 import autokitteh
 from autokitteh.google import google_calendar_client
-
-
-CALENDAR_CONNECTION_NAME = "my_google_calendar"
-JIRA_CONNECTION_NAME = "my_jira"
+from autokitteh.atlassian import get_url
```
[the remark this answers is not in the document]
Comment (pashafateev) [reply jira_to_google_calendar/program.py]: Done

File: jira_to_google_calendar/program.py (hunk 1 above)
Comment (pashafateev) [reply jira_to_google_calendar/program.py:24]: Done

**Pull request merged**